backend/Makefile

```
@@ -60,6 +60,20 @@ fmt:
 	@echo ">>> gofumpt -w ."
 	@$(GOFUMPT) -w .
 
+# repository interface(port) から usecase テスト用の fake を生成する(手書き fake の代替)。
+# 生成物 internal/usecase/repository/repofakes/fakes_gen.go はコミット対象。
+# port にメソッドを追加/変更したら必ず `make fakegen` で再生成する。
+# 生成後に gofumpt をかけて CI の整形チェックを通す。
+.PHONY: fakegen
+fakegen:
```

checkmake (0.3.2)

[warning] Target body for "fakegen" exceeds allowed length of 5 lines (7).

(maxbodylength)